src/Pos/Script/Examples.hs

```diff
@@ -141,7 +141,8 @@ multisigRedeemer txSigData sks = fromE $ parseRedeemer Nothing [text|
     mkCons Nothing s = sformat ("(Cons Nothing "%build%")") s
     mkCons (Just sig) s = sformat
         ("(Cons (Just #"%fullSignatureHexF%") "%build%")") sig s
-    shownSigs = foldr mkCons "Nil" $ (`safeSign` txSigData) <<$>> sks
+    sigs = map (fmap (\k -> safeSign SignTxIn k txSigData)) sks
```

Comment:
Why don't you like `<<$>>` here? Though maybe we should add `<<&>>` operator into `universum`...